src/plugins/chromecast/style.css

```
@@ -0,0 +1,96 @@
+  transition: color 0.15s ease, background 0.15s ease;
+}
+
+.chromecast-button:hover {
+  color: #fff;
+  background: rgba(255, 255, 255, 0.1);
+}
+
+.chromecast-button svg {
+  width: 24px;
+  height: 24px;
+  fill: currentColor;
```

Stylelint (17.12.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)